src/scss/grommet-core/_objects.carousel.scss

```diff
@@ -20,8 +20,8 @@
     max-width: none;
     transition: all 0.8s;
 
-    > * {
-      flex: 1;
+    .tiles > .tile > * {
```

Comment:
can we add a class for these tiles? like "carousel_item". In the future if we decide to move away from tiles we would not need to change the css.